core/platform/search/gae_search_services.py

```diff
@@ -118,6 +130,20 @@ def _dict_to_search_document(d):
 
 
 def _make_fields(key, value):
+    """Returns the fields corresponding to the key value pair according to the
+    type of value.
+
+    Args:
+        key: str. The name of the field.
+        value: str. The field value.
```

Comment:
shouldn't this be of type *? It seems to take a lot more than str.

Reply:
Yeah, I rechecked. Done!